docs/workflow/sdk-web-create-element.mdx

```diff
@@ -30,37 +30,28 @@ In the Geocortex Workflow TypeScript SDK, form elements are represented by TypeS
 ```tsx
 import * as React from "react";
 import {
```

Comment:
Should we keep the old instructions somewhere or have some kind of migration guide? I realize the blog post contains everything, so maybe we could link to that, or have it in a separate page. I'm just thinking of the experience of a developer who used the old API and then came to this page only to find the code is totally different.

Reply:
We don't currently have versioned docs, nor did we have any desire to do so. The idea was that we provide the "latest" version for everything. It's open source so they could freely look back in the history to see what the old info was if they really needed to.

The previous (current) forms API is intended to be in a deprecated state once this is released. New elements should use the new patterns.